packages/@glimmer/global-context/index.ts

```diff
@@ -14,6 +14,13 @@
  */
 
 import { DEBUG } from '@glimmer/env';
+import { assign } from '@glimmer/util';
+
+//////////
+
+export const FEATURES = {
+  DEFAULT_HELPER_MANAGER: false,
+};
```

Comment:
Let's make this a top level `let` variable:

```suggestion
export let FEATURE_DEFAULT_HELPER_MANAGER = false;
```

The `const` here only mean the object cannot be reassigned, but the POJO can be mutated by whoever imports it, which introduces a "backdoor" for changing the value of the flags without going through the `setGlobalContext` API. (Using an object in that API is fine though.)

Reply:
Also let's default this to `true` by default.